modules/nw-udn-best-practices.adoc

```diff
@@ -0,0 +1,33 @@
+//module included in the following assembly:
+//
+// *networkking/multiple_networks/understanding-user-defined-networks.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="considerations-for-udn_{context}"]
+= Best practices for UserDefinedNetwork
+
+Before setting up a `UserDefinedNetwork` (UDN) resource, users should consider the following information:
+
+* To eliminate errors and ensure connectivity to services, you should create a namespace scoped UDN CR, create pods in the namespace, and then create services in that namespace.
+
+* `Openshift-*` namespaces should not be used to set up a UDN.
+
+* You might want to allow access to any Kubernetes services on the cluster default  network. By default, KAPI and DNS are accessible.
+* You must reconfigure your masquerade subnet to be large enough to hold the required number of networks. For more information, see "Understanding the Cluster Network Operator".
+// May be something that is downstream only.
+//* No active primary UDN managed pod can also be a candidate for `v1.multus-cni.io/default-network`
+
+* Ensure tenants are using the `UserDefinedNetwork` resource and not `network-attachement-definitions` resource. This can create security risks between tenants.
+
+* When configuring pod IP addresses the CIDR cannot overlap with the default cluster CIDR used by the cluster default network. OVN-Kubernetes network plugin uses the following address for the default network: `100.64.0.0/16`, `169.254.169.0/29`, `100.88.0.0/16`, `fd98::/64`, `fd69::/125`, and `fd97::/64`.
++
+[IMPORTANT]
+====
+For {product-title} 4.17 and later versions, clusters use `169.254.0.0/17` for IPv4 and `fd69::/112` for IPv6 as the default masquerade subnet. These ranges should also be avoided by users. For upgraded clusters, there is no change to the default masquerade subnet.
+====
+
+* The default network’s `PodSubnets` and `JoinSubnets` fields cannot be reused for any UDN.
+
+* You should not change pod IPAM annotations.
```

Comment:
I am not sure I understand this point as well, could you tell me what you meant?
I think you mean if you use CRs then we don't need the NSE elements ?

Reply:
From the corner cases docs:
`Don’t touch pod IPAM annotations (not UDN specific)
Why? CNI side uses it for CNI Add; worst case is a wrongly programmed podIP` 

I think we can just delete, right?